cmd/deploy.go

```diff
@@ -0,0 +1,159 @@
+package cmd
```

Comment:
We should create component here. deploy branches does too much.
the command should only get the argument needed to call the components...
We should separate the ux from the business code...